dashboard/config/libraries/NativeSpriteLab.interpreted.js

```diff
@@ -0,0 +1,171 @@
+function draw() {
+  executeDrawLoopAndCallbacks();
+}
+
+/* Legacy code only. Do not add any new code below here */
+function clickedOn(spriteId, callback) {
+  spriteClicked('when', spriteId, callback);
+}
+
+function draggable() {
+  return {func: draggableFunc(), name: 'draggable'};
+}
+
+function pointInDirection(spriteId,direction) {
+  if (direction== "North") {
+    setProp(spriteId, 'rotation', 360);
+  }
+  else if (direction== "East") {
+    setProp(spriteId, 'rotation', 90);
+  }
+  else if (direction=="South") {
+    setProp(spriteId, 'rotation', 180);
+  }
+  else if (direction=="West") {
+    setProp(spriteId, 'rotation', 270);
+  }
+  else {
+    console.error("pointInDirection: invalid direction provided");
+  }
+}
+
+function randColor() {
+  return color(randomNumber(0, 255), randomNumber(0, 255), randomNumber(0, 255)).toString();
+}
+
+function randomColor() {
+  return randColor();
```

Comment:
Do both randColor and randomColor get used?

Reply:
Yes, I think so. Or at least, we have no way to be really sure which function is used where, so the safest option is to just keep them both.

Reply:
haha 🤦‍♂ sounds good